bootstrap/src/lib/cache/index.ts

```diff
@@ -150,8 +150,6 @@ export const withCache: Middleware = async (execute, options = defaultOptions())
 
     if (entry) {
       if (maxAge >= 0) {
-        logger.debug(`Cache: GET ${key}`, entry)
```

Comment:
Why did you remove this? I think we'd need this feature to update the already cached item with the new TTL.

If you don't want for warmer to recache the item, then don't set a `maxAge` to the input and that will achieve the same thing.

Reply:
This is keeping the cache values forever. As maxAge from the input is constantly changing, it updates the cache entry with the same values, but with extended maxAge

Reply:
I've included the max age coming from the RateLimit middleware into `data: { rateLimitMaxAge: ... }`. If the request contains `maxAge` would override the default value